src/frontend/src/components/SearchResults/Case.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+interface Props {
+  case: {
+    name: string;
+    case_number: string;
+    birth_year: number;
+    balance_due: number;
+  };
+}
```

Comment:
This is essentially a type definition for a Case, and so it should probably be in recordexpungPDX/src/frontend/src/redux/records/types.ts

Reply:
Done